Obliquity model - Issue 34

klaferr opened the pull request:
- Added a linear model that uses obliquity instead of insolation in accumulation model
- added tests to related file
- added a statement into trough.py to input the right values for obliquity instead of insolation 
- updated test notebook to show the result of using obliquity dependence

I also added a new file for obliquity which is what I interpreted as the time steps since the original obliquity.txt file did not contain the time steps. 

Closes #34

tmcclintock commented on mars_troughs/data/Obliquity_new.txt:
```diff
@@ -0,0 +1,1002 @@
+Obliquity (units?) Time(yrs)
```
I'm pretty sure the units of obliquity are degrees. You should double check with @ambramson on this though and also what the times are. @ambramson made the original obliquity text file.

tmcclintock reviewed (changes requested):
Looks great so far. Just a few minor things.

Also, can you add another test to `test_trough.py` where we instantiate a `Trough` with an obliquity model? We should just make sure it all works OK.

klaferr commented:
With new pushes:
 - moved get_accumulation_at_t to timedependentmodel
 - fixed docstrings
- fixed naming scheme for model
- added optional type for model names in trough.py
- fixed obliquity file header
I wasn't sure what you wanted by a test for the obliquity model but I added a test that I think test the set up of an obliquity model-based trough

tmcclintock commented:
@klaferr looks great to me. Nice job! Feel free to merge to main.
